integration_tests/pages/components/contact-information/contactDetailsForm.ts

```diff
@@ -0,0 +1,49 @@
+import { PageElement } from '../../page'
+import FormInputComponent from '../formInputComponent'
+
+export type ContactDetailsFormData = {
+  contactNumber?: string
+}
+
+export default class ContactDetailsFormComponent {
+  private get form(): PageElement {
+    return cy.get('form') // [action*="contact-information/contact-details"]
+  }
+
+  checkHasForm(): void {
+    this.form.should('exist')
+  }
+
+  hasAction(action: string | RegExp): PageElement {
+    return this.form.should('have.attr', 'action', action)
+  }
```

Comment:
I think every form has these (or at least most) so maybe we abstract these to a base class?